website_form_recaptcha/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016-2017 LasLabs Inc.
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+
+<odoo>
+    <data noupdate="1">
+        <record id="recaptcha_key_site" model="ir.config_parameter">
+			<field name="key">recaptcha.key.site</field>
+			<field name="value">Obtain from Google</field>
+        </record>
+		<record id="recaptcha_key_secret" model="ir.config_parameter">
+			<field name="key">recaptcha.key.secret</field>
+			<field name="value">Obtain from Google</field>
```

Comment:
[You shouldn't remove the groups line](https://github.com/OCA/website/pull/395/commits/ec1240455f9b4ba1c51b922eff0a3cc96a9621ed#diff-567885b3b5da5f9ff0f0bdf646534a4fL20), this secret must not be world-readable.

Reply:
The `group_ids` field is gone as of odoo/odoo@d0ca2d115e8b40308e1ff83bf9e7cc874762fbff, every `ir.config_parameter` must be accessed with `sudo()` now.

Reply:
Thanks for the info. Added to the migration wiki.